documents/Libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -43,6 +43,17 @@
   <typedef name="stringarray"/>
   <typedef name="geomnamearray"/>
 
+  <!-- ======================================================================== -->
+  <!-- Geomprop definitions                                                     -->
+  <!-- ======================================================================== -->
+
+  <geompropdef name="Pobject" node="position" type="vector3" space="object"/>
+  <geompropdef name="Nobject" node="normal" type="vector3" space="object"/>
+  <geompropdef name="Tobject" node="tangent" type="vector3" space="object"/>
+  <geompropdef name="Pworld" node="position" type="vector3" space="world"/>
+  <geompropdef name="Nworld" node="normal" type="vector3" space="world"/>
+  <geompropdef name="Tworld" node="tangent" type="vector3" space="world"/>
+  <geompropdef name="UV0" node="texcoord" type="vector2" index="0"/>
```

Comment:
Define the commonly used geomprops in the std library.

Reply:
Looks like bitangent is also a default. I'm wondering if color should also be included here ?

Reply:
I only added the ones we currently use in the std and pbr libraries (used as default fallback on our nodedef inputs). We could add more but they would not be used until we add in nodedefs that use them, or users create their own nodedef libraries.

Bitangent might make sense to add though (even if rarely used) just to cover all the basic shading context data. I can add that in. For color data I think it's even more rarely used as a default fallback connection on nodedefs. So maybe wait with that one.

For creators of custom libraries it's always possible to create their own geompropdefs if they need a default connection to a particular data that we have not supplied out of the box.